src/components/PostEditor/PostEditor.js

```diff
@@ -544,17 +542,21 @@ export default class PostEditor extends React.Component {
 
     const hasStripeAccount = get('hasStripeAccount', currentUser)
     const hasLocation = [
+      'discussion',
```

Comment:
well we never officially decided this is what we want to do, but i like it, i will double check with the rest of the team